demo/admin/crud-generator-config.ts

```diff
@@ -8,4 +8,18 @@ export default [
         target: "src/news/generated",
         entityName: "News",
     },
+    {
+        target: "src/books/generated",
+        entityName: "Book",
+        // if your entity expects block JSON for some props, you might need to explicitly define rootBlocks
+        // that's not necessary if the used block is defined
+        //   - in src/common/blocks and conforms to our naming conventions
+        //   - in a blocks folder right next to the generated folder and conforms to our naming conventions
+        //   - in @comet/cms-admin or @comet/blocks-admin
```

Comment:
@nsams Is that correct now?

Reply:
yes; would write this into the docs though

Reply:
Sure, still this PR should serve as a simple tutorial on how to use the admin generator. So it's good to have it here too.